integration_tests/cli/upgrade-demo.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Copyright 2020 DSR Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+source integration_tests/cli/common.sh
+
+upgrade_height=$RANDOM
```

Comment:
$RANDOM returns a random integer in the range from 0 to 32767. If it returns too small value then upgrade procedure can be started during a test execution. This will break the pool and thus fail the test.

So either a quite big fixed height should be used or a quite big number should be added to the random value.

Reply:
fixed